modules/ROOT/pages/indexes/semantic-indexes/vector-indexes.adoc

```diff
@@ -1,503 +1,302 @@
+| "Godfather: Part II, The"  | "The early life and career of Vito Corleone in 1920s New York is portrayed while his son, Michael, expands and tightens his grip on his crime syndicate stretching from Lake Tahoe, Nevada to pre-revolution 1958 Cuba." | 0.9547788500785828 |
+| "Scarface"                 | "An ambitious and near insanely violent gangster climbs the ladder of success in the mob, but his weaknesses prove to be his downfall."                                                                                  | 0.9367183446884155 |
+| "Jane Austen's Mafia!"     | "Takeoff on the Godfather with the son of a mafia king taking over for his dying father"                                                                                                                                 | 0.9366795420646667 |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 ----
 
-.Result
-[role="queryresult",options="header,footer",cols="3m,2m,2m,3m,3m,6m"]
-|===
+Note that all movies returned have a plot centred around criminal family organizations.
+The `score` results are  returned in _descending score order_, where the best matching result entry is put first (in this case, `The Godfather` has a similarity score of `1.0`, which is to be expected as the index was queried with this specific property).
+If the query vector itself is not wanted, adding the predicate `WHERE score < 1` removes identical vectors.
 
-| name | type | entityType | labelsOrTypes | properties | options
+To query a relationship vector index, use the link:{neo4j-docs-base-uri}/operations-manual/{page-version}/reference/procedures/#procedure_db_index_vector_queryRelationships[`db.index.vector.queryRelationships`] procedure.
 
-| "abstract-embeddings" | "VECTOR" | "NODE" | ["Abstract"]  | ["embedding"]
-| {indexProvider: "vector-2.0", indexConfig: {vector.dimensions: 1536, vector.similarity_function: "cosine"}}
+.Signature for `db.index.vector.queryRelationships` label:new[Introduced in 5.18]
+[source,syntax]
+----
+db.index.vector.queryRelationships(indexName :: STRING, numberOfNearestNeighbours :: INTEGER, query :: ANY) :: (relationship :: RELATIONSHIP, score :: FLOAT)
+----
 
-| "review-embeddings" | "VECTOR" | "RELATIONSHIP" | ["REVIEWED"]  | ["embedding"]
-| {indexProvider: "vector-2.0", indexConfig: {vector.dimensions: 256, vector.similarity_function: "cosine"}}
-5+d|Rows: 2
+[TIP]
+To compute the similarity score between two specific vector pairs, without using a vector index, use xref:functions/vector.adoc[].
 
-|===
+=== Performance suggestions
```

Comment:
I can understand moving this, but why is this a subsection under Query?
This is a general performance improvement in both index creation _and_ querying.